src/__tests__/lib/utils/SynapseClient.test.tsx

```diff
@@ -106,30 +86,10 @@ describe('it works at integration level testing', () => {
       ],
     }
     return SynapseClient.getFiles(request)
-      .then((data) => {
+      .then(data => {
         expect(data.requestedFiles).toBeDefined()
       })
-      .catch((err) => {
-        fail(err.reason)
-      })
-  })
-
-  it('get entity bundle latest version', () => {
-    const partsMask =
-      SynapseConstants.ENTITY_BUNDLE_MASK_ENTITY |
-      SynapseConstants.ENTITY_BUNDLE_MASK_RESTRICTION_INFORMATION |
-      SynapseConstants.ENTITY_BUNDLE_MASK_FILE_HANDLES
-    return SynapseClient.getEntityBundleForVersion(
-      'syn1725696',
-      undefined,
-      partsMask,
-    )
-      .then((data) => {
-        expect(data.entity).toBeDefined()
-        expect(data.restrictionInformation).toBeDefined()
-        expect(data.fileHandles).toBeDefined()
-      })
-      .catch((err) => {
+      .catch(err => {
         fail(err.reason)
       })
   })
```

Comment:
This API is deprecated, so why test it

Reply:
That's reasonable.  Are we using the updated API (entity bundle2 services)?  Is there a test?

Reply:
We have a call to SynapseClient that uses the v2 API. Like most of the SynapseClient functions, we don't have an explicit test for it. The Entity Finder is one example of a component that is using it.